src/annotator/util/frame-util.js

```diff
@@ -43,6 +43,16 @@ function isValid (iframe) {
   return iframe.className !== 'h-sidebar-iframe';
 }
 
+function isReady (iframe, callback) {
```

Comment:
Not a show stopper for this PR but it's not obvious the difference/needs of the isReady function vs the isLoaded. Perhaps we can change the names (or one of them) to something less abstract if it's trying to guarantee some sort of load state